Kentico.Kontent.Management/Models/Types/Elements/ValidationRegexModel.cs

```diff
@@ -14,5 +16,11 @@ public class ValidationRegexModel
         /// Specifies regular expression flags that affect the search. Supports only case-insensitive therefore only allowed flag is 'i'.
         /// </summary>
         public string Flags { get; set; }
+        
+        /// <summary>
+        ///  Specifies the custom message that is used when 
```

Comment:
This sentence seems incomplete. When is a custom message used?

Reply:
Sorry. Should be okay now :)